lib/smart_answer_flows/inherits-someone-dies-without-will.rb

```diff
@@ -111,10 +111,6 @@ def define
         save_input_as :parents
 
         on_condition(variable_matches(:region, 'england-and-wales')) do
-          on_condition(variable_matches(:partner, 'yes')) do
-            next_node_if(:outcome_21, responded_with('yes'))
-            next_node_if(:siblings?, responded_with('no'))
-          end
           on_condition(variable_matches(:partner, 'no')) do
```

Comment:
Ah. I see. I had misunderstood. I don't see a problem with removing the `on_condition` as you've described.